### PR TITLE
etcdctl: list output options

### PR DESCRIPTION
**Pull request by xiang90**
Fix https://github.com/coreos/etcd/issues/6233

/cc @gyuho @heyitsanthony

### CODE REVIEW

**Comment by gyuho** [comment]
LGTM. Thanks!

File: etcdctl/ctlv3/ctl.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ var (
 func init() {
 	rootCmd.PersistentFlags().StringSliceVar(&globalFlags.Endpoints, "endpoints", []string{"127.0.0.1:2379"}, "gRPC endpoints")
 
-	rootCmd.PersistentFlags().StringVarP(&globalFlags.OutputFormat, "write-out", "w", "simple", "set the output format (simple, json, etc..)")
+	rootCmd.PersistentFlags().StringVarP(&globalFlags.OutputFormat, "write-out", "w", "simple", "set the output format (simple, json, proto, table)")
```
Comment (heyitsanthony) [inline etcdctl/ctlv3/ctl.go]: alphabetize?

**Comment by heyitsanthony** [comment]
lgtm

File: etcdctl/ctlv3/ctl.go (hunk 1 above)
Comment (xiang90) [inline etcdctl/ctlv3/ctl.go]: fixed.

**Pull request merged**